src/panels/lovelace/cards/hui-spacing-card.ts

```diff
@@ -0,0 +1,36 @@
+import type { PropertyValues } from "lit";
+import { LitElement, html } from "lit";
+import { customElement } from "lit/decorators";
+import "../../../components/ha-card";
+import { hasConfigOrEntityChanged } from "../common/has-changed";
+import type { LovelaceCard, LovelaceCardEditor } from "../types";
+
+@customElement("hui-spacing-card")
+export class HuiSpacingCard extends LitElement implements LovelaceCard {
+  public static async getConfigElement(): Promise<LovelaceCardEditor> {
+    await import("../editor/config-elements/hui-spacing-card-editor");
+    return document.createElement("hui-spacing-card-editor");
+  }
+
+  protected shouldUpdate(changedProps: PropertyValues): boolean {
+    return hasConfigOrEntityChanged(this, changedProps);
+  }
+
+  public getCardSize(): number {
+    return 1;
+  }
+
+  public setConfig(): void {
+    // No config necessary
+  }
+
+  protected render() {
+    return html`<ha-card></ha-card>`;
```

Comment:
Why do we use `ha-card` and not just a `div`, or set the `:host { display: block; }`?